tests/inject/utils/url.tests.ts

```diff
@@ -1,7 +1,9 @@
-import {isURLEnabled, isURLMatched, isPDF, isFullyQualifiedDomain, getURLHostOrProtocol, getAbsoluteURL} from '../../src/utils/url';
-import type {UserSettings} from '../../src/definitions';
+// noinspection HttpUrlsUsage
```

Comment:
??

Reply:
I guess `inject` should be changed as it rather should reflect it includes DOM actions etc.

Reply:
Hm, I must have allowed IntelliJ to add that in. I'll remove it.

Reply:
> I guess inject should be changed as it rather should reflect it includes DOM actions etc.

Not sure what you mean, sorry.

Background of this change: I used to be a Jest test, and it was only working because we were using jsdom as the runner. When we removed that, I moved the test into the Karma test suite because it depends on the DOM. 

This is the original discussion:

https://github.com/darkreader/darkreader/pull/8266#discussion_r813307271

Reply:
I referred to the folder structure.

Reply:
It needs DOM:

https://github.com/darkreader/darkreader/blob/master/src/utils/url.ts#L9-L15

I thought [you said that](https://github.com/darkreader/darkreader/pull/8122#issuecomment-1030800631) the ./inject directory was for browser/DOM tests? That's how I've been picturing it. Isn't `./inject` going to eventually replace `./browser`?

Reply:
Correctly, which is why I find the inject folder name to be quite weird. Just a thought, not really a issue with this PR.

Reply:
> ??

Fixed.